presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/RowExpressionVerifier.java

```diff
@@ -115,6 +116,23 @@ protected Boolean visitNode(Node node, RowExpression context)
         throw new IllegalStateException(format("Node %s is not supported", node));
     }
 
+    @Override
+    protected Boolean visitArrayConstructor(ArrayConstructor node, RowExpression context)
+    {
```

Comment:
What is this change for? Maybe move it to a different PR (or a different commit) if it's unrelated.

Reply:
It's for the expression `array[0.1, 0.2]` in test SQL, otherwise it will throw exception as `visitArrayConstructor` is not implemented.